mongo/entrypoint-initdb.d/init.js

```diff
@@ -0,0 +1,13 @@
+var databases = [
+    {
+        name: "interactives",
```

Comment:
i think `interactives` and `visualisations` are synonymous - iirc matt said on a call we should use interactives - but im not 100%, def worth checking 😊. if so, i think we should just name things interactives - not use both, just for clarity & consistency.